PowerShellTools/Intellisense/IntelliSenseManager.cs

```diff
@@ -78,18 +79,25 @@ public int Exec(ref Guid pguidCmdGroup, uint nCmdId, uint nCmdexecopt, IntPtr pv
         {
             if (VsShellUtilities.IsInAutomationFunction(_serviceProvider) ||
                 pguidCmdGroup != VSConstants.VSStd2K ||
-                Utilities.IsInCommentArea(_textView.Caret.Position.BufferPosition.Position, _textView.TextBuffer))
+                Utilities.IsInCommentArea(_textView.Caret.Position.BufferPosition.Position, _textView.TextBuffer) ||
+                IsUnrecognizedCommand(nCmdId))

```

Comment:
should we combine the cmdgroup Check into the IsUnrecognizedCommand [nit: rename to IsUnhandledCommand? We recognize it, but don't handle it :)]